articles/users/normalized/auth0/normalized-user-profile-schema.md

```diff
@@ -18,11 +18,11 @@ The attributes that Auth0 maps to a common schema are listed below.
 Fields that are always generated:
 
 * **`name`**: the user's full name.
-* **`nickname`**: the user's username.
+* **`nickname`**: the first part of the user's email, or the user's username if using the Auth0 data store and require username.
```

Comment:
```suggestion
* **`nickname`**: the user's username or the local part of the user's email.
```

Reply:
Are there conditions surrounding when each is used?

Reply:
No, the nickname default value is:
We first try to use username, but if username is not present, we set the local part of the email.
That's why I used the `OR` there.

Reply:
Updated. Thank you.